verilog/tools/formatter/README.md

```diff
@@ -74,6 +74,8 @@ To pipe from stdin, use '-' as <file>.
     --try_wrap_long_lines (If true, let the formatter attempt to optimize line
       wrapping decisions where wrapping is needed, else leave them unformatted.
       This is a short-term measure to reduce risk-of-harm.); default: false;
+    --wrap_end_else_clauses (Split end and else keywords into separate lines);
+      default: false; currently: true;
```

Comment:
Where is the `curerently: true;` text annotation coming from ? Were you passing the flag at the time you created the --helpfull output ?